music-theory-spectrum.csl

```diff
@@ -391,13 +391,13 @@
     <layout suffix="." delimiter="; ">
       <choose>
         <if position="ibid-with-locator">
-          <group delimiter=", ">
+          <group delimiter=" " prefix="(" suffix=")">
```

Comment:
@damnation333, I think the parentheses belong on the `<text macro="point-locators-subsequent"/>` to get "Ibid. (187-8)." Currently the "ibid" term would be wrapped as well.

I also don't think you need the parentheses on the "ibid" term below.